pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1410,8 +1410,18 @@ func (n *NetworkPolicyController) getAppliedToWorkloads(g *antreatypes.AppliedTo
 		group, found, _ := n.internalGroupStore.Get(g.SourceGroup)
 		if found {
 			grp := group.(*antreatypes.Group)
+			var nodes []*v1.Node
+			if grp.Selector != nil {
+				if grp.Selector.NodeSelector != nil {
+					nodes, err := n.nodeLister.List(grp.Selector.NodeSelector)
+					if err != nil {
+						nodes = nil
+					}
+					return nil, nil, nodes, err
+				}
+			}
```

Comment:
```suggestion
			if grp.Selector != nil && grp.Selector.NodeSelector != nil {
					nodes, err := n.nodeLister.List(grp.Selector.NodeSelector)
					return nil, nil, nodes, err
				}
			}
```
Note that the `nodes` defined via `var` was never used as it declared another `nodes` via `:=`.